version/version.go

```diff
@@ -1,5 +1,6 @@
 package version
 
+// TODO: remove this comment
```

Comment:
yes

Reply:
hahaha got catch, I did this just to push a commit and forgot about it